modules/claims_api/app/clients/claims_api/bgs_client/definitions.rb

```diff
@@ -175,6 +175,38 @@ module FindPersonBySSN
         end
       end
 
+      ##
+      # TrackedItemService
+      #
+      # Adding 'Bean' to the end to differentiate from the service
+      module TrackedItemServiceBean
+        DEFINITION =
```

Comment:
Should this be `TrackedItemService/TrackedItemService`?
<img width="443" alt="Screenshot 2024-08-28 at 1 04 38 PM" src="https://github.com/user-attachments/assets/c4f4fb93-64e7-41d1-96f2-dcdb5f4679bb">
<img width="614" alt="Screenshot 2024-08-28 at 1 05 05 PM" src="https://github.com/user-attachments/assets/e3b5c342-be87-4c84-b90b-eb54e9462165">
<img width="1168" alt="Screenshot 2024-08-28 at 1 11 12 PM" src="https://github.com/user-attachments/assets/1b0eea33-9dd7-47ac-8287-12c4b5724130">

Reply:
As opposed to `TrackedItemServiceBean`? Yes, it only has `Bean` in the class name since the service and bean have the same name, they'd conflict. It (should) return w/out the `Bean` bit when actually used.

Reply:
> As opposed to `TrackedItemServiceBean`? Yes, it only has `Bean` in the class name since the service and bean have the same name, they'd conflict. It (should) return w/out the `Bean` bit when actually used.

@FonzMP, @acovrig Seems risky since this is used by Claims Show, is there another way to test it?